Sources/PointFree/Episode/Episode.swift

```diff
@@ -80,7 +80,7 @@ As server-side Swift becomes more popular and widely adopted, it will be importa
   length: 1380,
   publishedAt: 1_497_960_000,
   sequence: 4,
-  subscriberOnly: false,
+  subscriberOnly: true,
```

Comment:
the episode page now correctly blocks the transcript when a non-subscriber is viewing a subscriber only page